src/domUpdates.js

```diff
@@ -1,18 +1,19 @@
 // Imports
+import { pageData } from './apiCalls';
 import {recipeData} from './data/recipes'
 import { ingredientsData } from './data/ingredients';
-import { searchRecipes } from './recipes';
-import { currentUser, pageData } from './apiCalls';
 import {
-  recipeGrid,
-  clickedRecipe,
-  tagArea,
+  recipeGrid, 
+  clickedRecipe, 
+  tagArea, 
```

Comment:
this is showing you you have some trailing spaces that should be deleted

Reply:
done